app/carbonapi/http_handlers.go

```diff
@@ -193,9 +193,18 @@ func (app *App) renderHandler(w http.ResponseWriter, r *http.Request) {
 	var targetErrs []error
 	size := 0
 	defer func() {
+		//TODO: cleanup RenderDurationPerPointExp
 		if size > 0 {
 			app.prometheusMetrics.RenderDurationPerPointExp.Observe(time.Since(t0).Seconds() * 1000 / float64(size))
 		}
+		//2xx response code is treated as success
+		if toLog.HttpCode > 0 && toLog.HttpCode/100 == 2 {
```

Comment:
I guess this can be replaced with just second part. The second part is false if the first one is false.

Reply:
I agree with @grzkv

Reply:
yes, it was added as just a safety check.